client/app/src/main/java/com/example/criengine/Database/DatabaseWrapper.java

```diff
@@ -323,6 +323,33 @@ public Boolean apply(@NonNull Transaction transaction) throws FirebaseFirestoreE
                 });
     }
 
+    /**
+     * Get a list of all books in the database
+     * @return list of all books
+     */
+    public Task<List<Book>> searchBooks() {
```

Comment:
At some point we also need this fragment to redirect to a username if a valid username is entered, but that's a later problem. This looks workable for now